src/utils.ts

```diff
@@ -60,8 +60,8 @@ export function greatestCommonDenominator(numerator: BigInt, denominator: BigInt
 }
 
 export function reduce(numerator: BigInt, denominator: BigInt): BigInt[] {
-  let greatestCommonDenominator = greatestCommonDenominator(numerator, denominator)
-  return [numerator.div(greatestCommonDenominator), denominator.div(greatestCommonDenominator)]
+  let gcd = greatestCommonDenominator(numerator, denominator)
```

Comment:
the lint was super smart here ;) 
he notice that `greatestCommonDenominator` is in the function, so it's good enough to have the acronym there

Reply:
It essentially complained about shadowing the function name since the variable and function had the same name .

Reply:
wow, so it was smart indeed! I thought it was u that spotted it 😂